src/java/map/SCMap.java

```diff
@@ -6,6 +6,7 @@
 import util.Vector4f;
 import util.serialized.WaterSettings;
 
+import java.awt.*;
```

Comment:
pls run optimize imports over the entire project (select src folder + press ctrl+alt+o)